locales/it/LC_MESSAGES/volto.po

```diff
@@ -1482,11 +1482,11 @@ msgstr "Titolo"
 
 #: config/Blocks
 msgid "toc"
-msgstr ""
+msgstr "Table of contents"
```

Comment:
Non c'è una frase in italiano che ci sta bene?

Reply:
Mi viene in mente solo "Indice dei contenuti"